Assigment2/Assignment2.java

```diff
@@ -0,0 +1,95 @@
+import java.util.*;
+import assignment2.*;
+public class Assignment2{
+    public static void main(String[] args) {
+        String choice = null;
+    
+        do{
+
+                try (Scanner sc = new Scanner(System.in)) {
+                    System.out.println();
+                    System.out.println("Types of Shape :\n\t1)Circle\n\t2)Recatangle\n\t3)Square\n\t4)Cube\n\t5)Triangle\n\t6)Sphere\n\t7)Cylinder\n\t8)exit the program\n\t");
+                    System.out.print("Enter your choice(in numbers): ");
+                    choice = sc.next();
+                }
+                switch (choice) {
+                    case "1":
+                        try {
+                            
+                            Cricle cricle = new Cricle();
+                            cricle.calculation();
+                        } 
+                        catch (Exception e) {
+                        
+                            System.out.println("invaild");
```

Comment:
Can you please add a proper message for invalid?

Reply:
> Can you please add a proper message for invalid?
Yes sir I specified each method .